controllers/healthcheck.go

```diff
@@ -0,0 +1,64 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/prest/prest/adapters/postgres"
+)
+
+type HealthCheck struct {
+	Status string `json:"status"`
+}
+
+// Created an interface for future mocking test
+type iDbConnection interface {
+	GetConnection() (*sqlx.DB, error)
+	RunTestQuery() error
+}
+
+type DbConnection struct{}
+
+func (d DbConnection) GetConnection() (db *sqlx.DB, err error) {
+	db, err = postgres.Get()
+	if err != nil {
+		return
+	}
+	return
+}
+
+func (d DbConnection) RunTestQuery() (err error) {
+	db, _ := d.GetConnection()
+	_, err = db.Exec("SELECT 1")
```

Comment:
```suggestion
	_, err = db.Exec(";")
```
Using just `;` is enough for this check and less expensive because it will not require parsing/planning. Another approach is to use `db.Ping()` (not sure but I guess behind the scenes it uses PQping or PQpingParams from libpqw - https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PQPINGPARAMS) and if it is true it is even less expensive for the backend.